lib/swagger-catalog.js

```diff
@@ -197,8 +224,10 @@ function getSchemesFromContext (nodeContext) {
 }
 
 function getHost(nodeContext) {
-  return nodeContext.https ? `${os.hostname()}:${nodeContext.https.port}`
-                           : `${os.hostname()}:${nodeContext.http.port}`;
+  let hostname = os.hostname();
```

Comment:
wasnt os.hostname a problem for us when it didnt match the fully qualified hostname value seen in ZOWE_EXPLORER_HOST?
it would be better to do
`let hostname = process.env('ZOWE_EXPLORER_HOST') ? process.env('ZOWE_EXPLORER_HOST') : os.hostname();`